build/images/scripts/install_cni

```diff
@@ -2,25 +2,46 @@
 
 set -euo pipefail
 
+# Fetching the list of the binaries that user wants to skip installing.
+arguments=$SKIP_CNI_BINARIES
+oldDelimeter=$IFS
+IFS=','
+flags=$arguments
+binaries=()
+for args in $flags; do
+  binaries+=($args)
+done
+IFS=$oldDelimeter
 # Todo: check version and continue installation only for a newer version
 
 # Install Antrea binary file
 install -m 755 /usr/local/bin/antrea-cni /host/opt/cni/bin/antrea
 
+# Checking the condition before installing the binaries, to ensure that user
+# does not want to skip the corressponding CNI binary file
+
 # Install the loopback plugin.
 # It is required by kubelet on Linux when using docker as the container runtime.
 # We replace the binary files even they are already present on the Node to make
 # sure expected versions are used.
-install -m 755 /opt/cni/bin/loopback /host/opt/cni/bin/loopback
+if [[ ! " ${binaries[*]}" =~ "loopback" ]]; then
```

Comment:
Although it works for current case, it's not very precise. For example, if there is one cni called loopback2, it would be matched by mistake. This would be more precise:
```
if [[ ! " ${binaries[*]} " =~ " loopback " ]]; then
```